app/src/main/res/layout/list_comments_item.xml

```diff
@@ -94,7 +94,7 @@
         android:contentDescription="@string/detail_heart_img_view_description"
         android:visibility="gone"
         tools:visibility="visible"
-        app:srcCompat="@drawable/ic_heart" />
+        android:src="@drawable/ic_heart" />
 
 
     <!--we can uncomment below code if we need dislike button and count in future-->
```

Comment:
I would also remove this. If in the future we need the dislike button, we will just copy-paste the XML for the like button, there is no need to keep some commented code that is probably old.